### PR TITLE
Upgraded lucas kanade optic flow module

### PR DESCRIPTION
**Pull request by hbrezak**
With new code optic flow results improved drastically. Test flown on ARdrone2 without any issue.

### CODE REVIEW

File: sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.c [hunk 1]
```diff
@@ -299,6 +300,10 @@ void calc_fast9_lukas_kanade(struct opticflow_t *opticflow, struct opticflow_sta
   float diff_flow_x = 0;
   float diff_flow_y = 0;
 
+  // Flow Derotation TODO:
```
Comment (OpenUAS) [inline sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.c]: Yeah the TODO,,,, was about the unknown impact, for me it is fine, all tests I've done were OK. Maybe K. McGuire has still something to add?

File: sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.c [hunk 2]
```diff
@@ -206,7 +212,7 @@ void calc_fast9_lukas_kanade(struct opticflow_t *opticflow, struct opticflow_sta
 
   // FAST corner detection (TODO: non fixed threshold)
```
Comment (OpenUAS) [inline sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.c]: Well ;) ... what about it?

**Comment by OpenUAS** [comment]
These can be left out of the pullrequest:

 conf/HB/hb_conf_speedy_opticflow.xml 
 conf/HB/hb_control_panel_speedy_opticflow.xml 
 conf/airframes/HB/hb_ardrone2_opticflowtest.xml 
 conf/flight_plans/HB/hb_cyberzoo.xml 

Header commet of 
 sw/airborne/modules/computer_vision/lib/vision/lucas_kanade.c 

Should be fixed, since the code was already existing, remove top part.

File: sw/airborne/modules/computer_vision/lib/vision/lucas_kanade.c [hunk 3]
```diff
@@ -48,31 +57,46 @@
...
-  //     [b] determine the image difference between the two neighborhoods
-  //     [c] calculate the 'b'-vector
-  //     [d] calculate the additional flow step and possibly terminate the iteration
+struct flow_t *opticFlowLK(struct image_t *new_img, struct image_t *old_img, struct point_t *points,
+                           uint16_t *points_cnt, uint16_t half_window_size,
+                           uint16_t subpixel_factor, uint8_t max_iterations, uint8_t step_threshold, uint8_t max_points, uint8_t pyramid_level)
+{
+
+
+  // Pyramidal implementation of Lucas-Kanade feature tracker.
```
Comment (flixr) [inline sw/airborne/modules/computer_vision/lib/vision/lucas_kanade.c]: would be nice to convert this into a doxygen comment for this function...

**Pull request merged**